charts/kube-state-metrics/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+name: kube-state-metrics
+description: Install kube-state-metrics to generate and expose cluster-level metrics
+keywords:
+- metric
+- monitoring
+- prometheus
+- kubernetes
+version: 2.9.2
+appVersion: 1.9.7
```

Comment:
We launched a pre-release candidate v2.0.0-alpha which is now merged in master, if we merge this it will differ.

Reply:
Hi @lilic. In general it's ok for the appVersion in a chart to be behind the newest release of the app it installs (for a short time anyway, it has to be, since the container image [tag](https://github.com/kubernetes/kube-state-metrics/pull/1237/files#diff-fd2bc408fdc516b05aa2e779a5417fb2R5) for the app must be available for CI to pass for the PR.

But in this case, 1.9.7 is the most recent release, right? https://github.com/kubernetes/kube-state-metrics/releases/tag/v1.9.7

Reply:
We have a pre-release of v2.0.0-alpha cut a few days ago, not sure if that counts for helm charts and the meaning of stable in helm? Just for your information https://github.com/kubernetes/kube-state-metrics/releases/tag/v2.0.0-alpha.